src/hub.html

```diff
@@ -1054,7 +1054,9 @@
                 rotation
                 set-yxz-order
                 pitch-yaw-rotator
-                matrix-auto-update></a-entity>
+                matrix-auto-update>
+                <a-sphere id="inspect-sphere"></a-sphere>
```

Comment:
i know we were burned by using primitives before in the teleporter stuff, is there any reason to be concerned here about poly count etc?

Reply:
Yes I want to replace this with a specific kind of "inspect mode environment object" that we like. At the very least, I'll create the THREE.js object directly instead of using this primitive.